### ISSUE TITLE
Attach uploads to specific dispositions

### ISSUE THREAD

**Issue by andycochran** [NYCPlanning/labs-zap-search#928]
For projects with multiple actions, depending on whether the user is submitting one or multiple recommendations, file uploads need to be attached to specific dispositions. 

When the user submits one recommendation for multiple actions, a copy of the uploaded file should be added to the Sharepoint folder for each action. _(When we display these files, they'll need to be de-duped, but we'll handle that in a separate issue.)_  
 
When the user submits multiple recommendations for each action separately, the file should be associated only with one particular action. This means that the recommendation form should be updated so that there's a file upload UI for each action when submitting multiple recommendations.

**Comment by hannahkates** [comment]
There is a potential enhancement in CRM that would identify a lead action. If that gets implemented, we could send all the files to the lead action disposition instead. We should wait on this until we can think more holistically about the desired work flow and document migration.
